mint/fakeAPIData.ts

```diff
@@ -203,17 +205,46 @@ export const wideData = [
     }
 ];
 
-export const columns =
-    {
-        "intervention": "Intervention",
-        "net_use": "Net use",
-        "prev_year_1": "Prevalence Under 5 yrs: Yr 1 post intervention",
-        "prev_year_2": "Prevalence Under 5 yrs: Yr 2 post intervention",
-        "prev_year_3": "Prevalence Under 5 yrs: Yr 3 post intervention",
-        "cases_averted": "Cases averted across 3 yrs since intervention",
-        "cases_averted_high": "Upper bound for cases averted",
-        "cases_averted_low": "Lower bound for cases averted"
-    };
+export const tableConfig: ColumnDefinition[] =
+    [
+        {
+            displayName: "Intervention",
+            valueCol: "intervention",
+            valueTransform: {
+                "none": "'none'",
```

Comment:
Note here the extra quotes around the name - this is because the mathjs evaluater will simply evaluate a string like this to itself.